src/openMVG/calibration/exportData.cpp

```diff
@@ -45,7 +49,15 @@ void exportImages(openMVG::dataio::FeedProvider& feed,
 
     openMVG::cameras::UndistortImage(inputImage, &camera, outputImage, openMVG::image::BLACK);
     const boost::filesystem::path imagePath = boost::filesystem::path(debugFolder) / (std::to_string(currentFrame) + suffix);
-    const bool exportStatus = openMVG::image::WriteImage(imagePath.c_str(), outputImage);
+
+#ifndef _WIN32
+	std::string nativePath = imagePath.native();
+#else
+	std::wstring_convert<boost::filesystem::path::codecvt_type> wsConv(&imagePath.codecvt());
+	std::string nativePath = wsConv.to_bytes(imagePath.native());
+#endif
+
+    const bool exportStatus = openMVG::image::WriteImage(nativePath.c_str(), outputImage);
```

Comment:
From the [documentation](http://www.boost.org/doc/libs/1_62_0/libs/filesystem/doc/reference.html):
`c_str()` is the same as `native().c_str()` which vary between linux and windows.

But `string()` always returns an std::string, so `string().c_str()` will always be the same.
`std::string    string(const codecvt_type& cvt=codecvt()) const;`